rs-commerce/src/App.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --font-family: 'Poppins', sans-serif;
+  --second-family: 'Montserrat', sans-serif;
```

Comment:
may be better to use name of font family in the name of variable? Easier to understand what font family it includes

Reply:
I renamed it